contracts/lib/ConsiderationDecoder.sol

```diff
@@ -909,11 +911,15 @@ contract ConsiderationDecoder {
                             add(totalOfferSize, totalConsiderationSize)
                         )
                         // Don't continue if returndatasize exceeds 65535 bytes
-                        // or is not equal to the calculated size.
+                        // or is greater than the calculated size.
                         invalidEncoding := or(
-                            gt(or(offerLength, considerationLength), 0xffff),
-                            xor(totalSize, returndatasize())
+                            gt(
+                                or(offerLength, considerationLength),
+                                generateOrder_maximum_returndatasize
+                            ),
+                            gt(totalSize, returndatasize())
```

Comment:
here we are allowing extra returned bytes than needed.

Reply:
That's consistent with the ABI standard--even though Seaport tries to be stricter at times.